src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.SnapshotManager.cs

```diff
@@ -54,7 +54,7 @@ private SnapshotManager(ImmutableArray<SharedWalkerState> walkerSharedStates, Im
                 int position,
                 BoundNode nodeToAnalyze,
                 Binder binder,
-                ImmutableDictionary<BoundExpression, (NullabilityInfo, TypeSymbol)>.Builder analyzedNullabilityMap,
+                ImmutableDictionary<BoundExpression, (NullabilityInfo, TypeSymbol?)>.Builder analyzedNullabilityMap,
```

Comment:
>TypeSymbol?) [](start = 71, length = 12)

This doesn't seem right. The builder won't ever let a null typesymbol be set, unless it's also missing annotations.

Reply:
Oh, sorry, misread this. Thought this was a different map.

---
In reply to: [452567880](https://github.com/dotnet/roslyn/pull/45640#discussion_r452567880) [](ancestors = 452567880)